### PR TITLE
Add thank you content for events module

### PR DESCRIPTION
**Pull request by lucymonie**
## What are you doing in this PR?
This adds some content about the events and masterclasses tickets to the digital subscriptions thank you page for the events AB test. 

In addition, this turns on the AB test in support-frontend by updating the abTestDefinitions test audience size to 1.

[**Figma**](https://www.figma.com/file/KNabD3Ns9G7SG9rXiBvX6r/Events?node-id=171%3A1126&viewport=1920%2C83%2C0.2595961093902588)
[**Trello Card**](https://trello.com/c/3FAknLiC/3774-ab-test-events-modules-thank-you-page-module)

## Why are you doing this?
To complete the work started in this PR: https://github.com/guardian/support-frontend/pull/3117

## Accessibility test checklist
 - [x] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
 - [x] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
 - [x] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

## Screenshots
support thegulocal com_subscribe_digital_checkout_promoCode=DK0NT24WG period=Monthly(wide) (3)

### CODE REVIEW

**Review by i-hardy — APPROVED** [review]
Looks great!

File: support-frontend/assets/pages/digital-subscription-checkout/components/thankYou/eventsModule.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+
+const EventsModule = () => (
+  <div>
+    <SvgTicket />
+    <Text title="Guardian digital events">
+      <p>
+      Enjoy 6 free tickets to Guardian digital events in the first 3 months of your subscription.
+      You will be sent your unique redemption code by email shortly, enter your code 6 times at
+      the checkout for your chosen events to receive your free tickets.
```
Comment (johnduffell) [inline support-frontend/assets/pages/digital-subscription-checkout/components/thankYou/eventsModule.jsx:24]:
```suggestion
      Enjoy 6 free tickets to Guardian digital events in the first 3 months of your subscription.
      You will be sent your unique redemption code by email shortly, which you can use 6 times at
      the checkout for your chosen events to receive your free tickets.
```

**Review by johnduffell — APPROVED** [review]
nice work 👍 just  atiny comment not a blocker, and I am guessing this copy was provided anyway!

File: support-frontend/assets/pages/digital-subscription-checkout/components/thankYou/eventsModule.jsx (hunk 1 above)
Comment (lucymonie) [reply support-frontend/assets/pages/digital-subscription-checkout/components/thankYou/eventsModule.jsx:24]: I think it's definitely an improvement @johnduffell but you would need to check this with Georgia Easteal as this text comes from her.

**Pull request merged**